colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -212,18 +213,19 @@
       "outputs": [],
       "source": [
         "run = wandb.init(project=\"artifact-basics\")\n",
-        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\") # selects the artifact you're adding the file to\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\") # selects the artifact you're adding the file to, and makes a new\n",
         "artifact.add_dir(local_path=\"/content/sample_data\", name=\"new_directory\")\n",
+        "run.log_artifact(artifact, aliases= [\"directory_version\"])\n",
```

Comment:
aliases typically represent state such as staging, prod, etc. can we make sure aliases listed reflect this